FrontEnd/src/components/ProfilePage/FormComponents/FormFields/HalfFormField.js

```diff
@@ -1,7 +1,14 @@
-import { PropTypes } from 'prop-types';
+import {PropTypes} from 'prop-types';
 import css from './HalfFormField.module.css';
+import {Tooltip} from 'antd';
+
+const LengthEmail = 32;
 
```

Comment:
The same as above. You should add spaces in the curly braces and rename the constant variable to upper case.